src/category_theory/single_obj.lean

```diff
@@ -64,46 +64,71 @@ protected def star : single_obj α := unit.star
 
 /-- The endomorphisms monoid of the only object in `single_obj α` is equivalent to the original
      monoid α. -/
-def to_End_equiv [monoid α] : End (single_obj.star α) ≃* α := mul_equiv.refl α
+def to_End [monoid α] : α ≃* End (single_obj.star α) :=
```

Comment:
Now that you're touching this code... shouldn't the equiv be the other way round? (The complicated side on the left...)?
I know that it doesn't really belong to this PR. Feel free to ignore.

Reply:
My idea was to use this equvalence to migrate some simple definitions from category theory to monoids. Then it makes sense to be able to write `to_End x`, where `x : α`, `[monoid α]`. Though we may have both `of_End` and `to_End = of_End.symm`.

P.S.: I wont' be able to update this PR in the next few days since I have no Internet at my new home yet. Writing this comment from a supermarket free wi-fi.

Reply:
Ok, that seems like a good application. Let's leave this as it is. We can always simply add `of_End = to_End.symm`